install_config/web_console_customization.adoc

```diff
@@ -505,6 +505,67 @@ ifdef::openshift-enterprise[]
 endif::[]
 ----
 
+[[system-status-badge]]
+=== System Status Badge
+
+The top navigation bar can also include an optional system status badge in order
+to notify users of system-wide events such as maintenance windows. To make use
+of the existing styles utilizing a yellow warning icon for the badge,
+follow the example below.
+
+. Create the configuration scripts within a file (for example,
+*_systemStatusBadge.js_*):
++
+[source, javascript]
+----
+'use strict';
+
+angular
+  .module('mysystemstatusbadgeextension', ['openshiftConsole'])
+  .run([
+    'extensionRegistry',
+    function(extensionRegistry) {
+      // Replace http://status.example.com/ with your domain
+      var system_status_elem = $('<a href="http://status.example.com/"' +
+      'target="_blank" class="nav-item-iconic system-status"><span title="' +
+      'System Status" class="fa status-icon pficon-warning-triangle-o">' +
+      '</span></a>');
+
+      extensionRegistry
```

Comment:
may want a comment here that says you should only add the extension point if you want the status icon to show up, a typical extension will optionally add this based on the existence of issues after hitting a status API.